cloudant-client/src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -188,8 +189,9 @@ public ApiKey generateApiKey() {
      * Get the list of all nodes and the list of active nodes in the cluster.
      *
      * @return Membership object encapsulating lists of all nodes and the cluster nodes
-     * @see <a target="_blank"
-     * href="https://docs.cloudant.com/advanced.html#get-/_membership">_membership</a>
+     * @see <a
+     * href="https://console.bluemix.net/docs/services/Cloudant/api/advanced.html#-get-_membership-">

```

Comment:
This link doesn't work although I'm not really sure why. Clicking the heading on the "advanced" page does produce this link and show the correct heading anchor. However, following this link directly does not appear to work. Perhaps there is a bug in the page?

Reply:
As discussed some experimentation with different browsers and whether or not other docs tabs are open suggests this is an issue in the docs pages rather than the links/anchors.

Reply:
I've opened FB ticket 95865.